### ISSUE TITLE
[FR] Config: replace hardcoded `Area` parameter by `xsetwacom --get N Area`

### ISSUE THREAD

**Issue by rubienr** [rubienr/wacom#2]
**Is your feature request related to a problem? Please describe.**

Configurations contain the input area parameter `Area` hard coded. 

**Describe the solution you'd like**

The device specific value is not always clear, neither do Wacom docs mention this specification.

**Describe alternatives you've considered**

Retrieve the values from the device on demand:

```bash
xsetwacom list
Wacom Cintiq 22HDT Pen stylus   	id: 11	type: STYLUS    
Wacom Cintiq 22HDT Pen eraser   	id: 14	type: ERASER    
Wacom Cintiq 22HDT Finger touch 	id: 10	type: TOUCH  

xsetwacom --set 10 ResetArea

xsetwacom --get 10 Area     
0 0 4752 2673
```

**Comment by rubienr** [comment]
Alternative:
- Input `Area` is taken as is from configuration to allow user defined clipping or calibration.
- Configured input `Area` can be overridden on demand with a script flag, example `xsetwacom.py --config ... device --map --override_config`

**Comment by rubienr** [comment]
Fixed in 5859fc0235613ce644524400a522b7e756f808bb.
